### PR TITLE
{bio}[GCCcore/12.2.0] dxpy v0.345.0 w/ Python 3.10.4 

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

Requires:
- [x] PR #17759

### CODE REVIEW

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Review by bedroge — DISMISSED** [review]
@sassy-crick A few more changes are required:
```
dxpy 0.345.0 requires argcomplete, which is not installed.
beautifulsoup4 4.12.2 requires soupsieve, which is not installed.
hatchling 1.14.0 has requirement packaging>=21.3, but you have packaging 20.9.
dxpy 0.345.0 has requirement websocket-client==0.54.0, but you have websocket-client 1.5.1.
```

**Comment by sassy-crick** [comment]
> @sassy-crick A few more changes are required:
> 
> ```
> dxpy 0.345.0 requires argcomplete, which is not installed.
> beautifulsoup4 4.12.2 requires soupsieve, which is not installed.
> hatchling 1.14.0 has requirement packaging>=21.3, but you have packaging 20.9.
> dxpy 0.345.0 has requirement websocket-client==0.54.0, but you have websocket-client 1.5.1.
> ```

Thanks for finding that. Somehow it installed ok on two different machines so I wrongly assumed it is all ok. 
I looked into it and found that updating the toolchain to `foss-2022b` solved some problems around `poetry` and thus I could fix the other version issues I had in the extension list. That unfortunately required to bump `FUSE` as well. 
I hope it is all working now.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node1 - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7763 64-Core Processor, Python 3.6.8
See https://gist.github.com/bedroge/3f5faab76b7c46f78417da8d7467dc68 for a full test report.

File: easybuild/easyconfigs/d/dxpy/dxpy-0.345.0-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+easyblock = 'PythonBundle'
+
+name = 'dxpy'
+version = '0.345.0'
+
+homepage = 'http://autodoc.dnanexus.com/'
+description = "DNAnexus Platform API bindings for Python"
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```
Comment (bedroge) [inline easybuild/easyconfigs/d/dxpy/dxpy-0.345.0-foss-2022b.eb:12]: This doesn't seem to use things like BLAS and MPI, so I guess we can lower the toolchain to GCCcore?

Comment (sassy-crick) [reply easybuild/easyconfigs/d/dxpy/dxpy-0.345.0-foss-2022b.eb]: Done.

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**